biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -192,8 +197,8 @@ module FatesPlantHydraulicsMod
   integer, public, parameter :: campbell_type           = 2
   integer, public, parameter :: tfs_type                = 3
   
-  integer, parameter :: plant_wrf_type = tfs_type
-  integer, parameter :: plant_wkf_type = tfs_type
+  integer, parameter :: plant_wrf_type = van_genuchten_type
```

Comment:
Is this set the default as van_genuchten? I would prefer to keep the default as TFS

Reply:
I will change default to be TFS

Reply:
done, confirmed